src/targeting/Find_targets.ml

```diff
@@ -36,6 +36,28 @@ module Out = Semgrep_output_v1_t
 
 module Fppath_set = Set.Make (Fppath)
 
+module Explicit_targets = struct
```

Comment:
I'm not a huge fan of nested modules. I'd rather move this code in a separate Explicit_targets.ml file.

Reply:
noted.